src/triggers.js

```diff
@@ -337,7 +337,16 @@ export function getResponseObject(request, resolve, reject) {
       }
       response = {};
       if (request.triggerName === Types.beforeSave) {
-        response['object'] = request.object._getSaveJSON();
+        try {
+          response['object'] = request.object._getSaveJSON();
```

Comment:
Are there any other possible error that may be silenced and overwritten by this general try/catch? Maybe we should check for the specific error in the catch block?

Reply:
I do not think there is any other error. And also no errors are not silenced.